test/rubygems/path.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'pathname'
+
+def source_root
```

Comment:
What about to keep it in module (similar to Bundler setup) to not "pollute" global (actually Kernel) "namespace"?